root/account/applications/Remove.js

```diff
@@ -7,8 +7,6 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-import * as React from 'react';
```

Comment:
This is unused because `ConfirmLayout` is called as a function below, like `ConfirmLayout({..})`, rather than as a React element like `<ConfirmLayout {...} />` (which would get transpiled into `React.createElement(ConfirmLayout, {...})`, so the import would be needed). Perhaps we should change it to use the React element syntax for consistency instead, but as long as we don't, well.

Reply:
Is there an actual benefit to calling it that way? (I guess consistency is already a benefit - in which case I could just make the change)

Reply:
I'm by no means a React expert, but I don't think it particularly matters unless we we're using performance tweaks like [React.PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent)/`shouldComponentUpdate` that involve the component lifecycle, or features like [error boundaries](https://reactjs.org/docs/error-boundaries.html).

Using React elements changes the "component tree," which can affect [reconciliation](https://reactjs.org/docs/reconciliation.html) too, but these components in particular are for static server-rendered pages which aren't being updated.

TL;DR: It doesn't matter here, and is probably slightly faster to call it as a function even, but using a React element would be more consistent.

Reply:
I'm changing them to be called as an element instead. Yay consistency!

Reply:
And for future use of error boundaries!